contributing/releasing.md

```diff
@@ -15,8 +15,8 @@ Each release is driven by a single **release engineer**, who is also a Googler.
 is expected to do the following, in order of priority:
 
 - Do not break Google.
-- Cut a release by first thing Wednesday morning, NYC time, without exception.
-- Land a release at least once a week.
+- Cut a release early in the working calendar week.
```

Comment:
Does this mean we're now allowed to delay a release to try and get a PR in? In the past that was "not allowed", but sounds like now it's OK?

Reply:
I still feel fairly strongly that when a release is cut should not be contingent on a given PR having landed; the release process is a train that should leave without conditions. Assuming we agree, how can we better clarify this intent in the wording here?

Reply:
I thought "as early in the week as possible" was good. The "as possible" was an important caveat to not doing it if it wasn't possible.

Reply:
@ianegordon Had expressed concern that Monday cuts aren't good. It may be worth getting his opinion on this shift.

Reply:
In my experience a lot of code lands on Monday.  Possibly from people not wanting to land code late on a Friday possibly to avoid breakages / rollbacks during the weekend.  Possibly from people taking off early on Friday and not being available for code reviews.  I can only guess.
Have we missed very many weekly releases due to Wednesday cuts not landing by Friday?

I wouldn't recommend cutting a release earlier than Tuesday morning.

Reply:
@ianegordon Do we have any historical data on changes that are intentionally landed on Mondays? 

I would suggest moving forward with this change (it seems to actually eliminate a hard requirement, not add one) and keep track of any instances where we are intentionally avoiding a Friday submit for specific reasons.  I haven't personally done any work specifically on a Friday in the past 18 months, and this change might just shift my expectation for releases from Wednesday morning to Monday morning.